### DISCUSSION TITLE
Updating Citizens' List

### DISCUSSION THREAD

**Discussion by marcalwestf** [adempiere/adempiere#4132] [ADempiere's New Launching]
This is a thread with the goal of updating the ADempiere citizens' list.

If you are new to the _Citizen_ concept, please follow the links at the end of this post so you can inform yourself about the matter.

If you know about citizens, rights, duties etc., you can suggest a new one or propose yourself. This can be done in a post in this very thread. The same applies for supporting candidatures.

You can also confirm your wish to keep your citizenship, if you are listed as an _active_ citizen. This can be done in a post in this thread.

Eventually, the citizens list will be updated and an official meeting will be summoned to decide important issues concerning the project. This will include founding projects, procedure to make repository changes (pull requests) etc.

**Links** 
Active Citizens: https://wiki.adempiere.net/Citizens

Deactivated Citizens: https://wiki.adempiere.net/Citizens_Inactive

Application Form:  https://www.adempiere.io/community/citizens/citizen-application-form.html

About Citizens: https://www.adempiere.io/community/community-detail.html

General Description about community: https://www.adempiere.io/community/

Citizens rights: https://www.adempiere.io/community/duties-and-rigths.html

**Comment by juddm** [comment]
Thanks for the post Mario - I would confirm I'd like to keep my citizenship :)

**Comment by armenrz** [comment]
Thanks Mario. Count me in.😊
